src/python/pants/backend/python/lint/black/rules.py

```diff
@@ -3,7 +3,7 @@
 
```

Comment:
This file and `isort/rules.py` were both greatly simplified to no longer use `BlackSetup`, `BlackArgs`, and `create_black_request()`. Now, we have one single `Setup`, `SetupRequest`, and `setup()` rule.

Both this and the Isort file are likely easier to review by looking at the entire file, rather than a unified diff.